packages/callout-block/src/CalloutBlock.tsx

```diff
@@ -73,6 +76,7 @@ export const CalloutBlock: FC<CalloutBlockProps> = ({ appBridge }) => {
     };
 
     const iconUrl = blockSettings.iconSwitch ? blockAssets?.[ICON_ASSET_ID]?.[0]?.genericUrl : '';
+    const hasNoIcon = blockSettings.iconType === Icon.None || (blockSettings.iconType === Icon.Custom && !iconUrl);
```

Comment:
Untested, but `blockSettings.iconType` seems to be undefined when a new block is created so you would also need to check for that:
```suggestion
    const hasNoIcon = !blockSettings.iconType || blockSettings.iconType === Icon.None || (blockSettings.iconType === Icon.Custom && !iconUrl);
```

Currently there is an empty space where the icon should be if you create a new block. If you then switch to Icon "none" the space disappears.

Or just set the default iconType to None.

Reply:
True! Thanks for checking again! Fixed